### PR TITLE
[6.4.0] Update module versions for release

### PR DESCRIPTION
**Pull request by cormacpayne**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/806/

File: tools/StaticAnalysis/BreakingChangeAnalyzer/BreakingChangeAnalyzer.cs [hunk 1]
```diff
@@ -166,7 +166,7 @@ public void Analyze(IEnumerable<string> cmdletProbingDirs, IEnumerable<string> m
                                     Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().CodeBase).AbsolutePath);
 
                                 string filePath = executingPath + "\\SerializedCmdlets\\" + fileName;
-                                bool serialize = false;
+                                bool serialize = true;
```
Comment (cormacpayne) [inline tools/StaticAnalysis/BreakingChangeAnalyzer/BreakingChangeAnalyzer.cs]: Will revert this change

File: tools/StaticAnalysis/Program.cs [hunk 2]
```diff
@@ -29,8 +29,8 @@ public class Program
     {
         static IList<IStaticAnalyzer> Analyzers = new List<IStaticAnalyzer>()
         {
-            new DependencyAnalyzer.DependencyAnalyzer(),
-            new SignatureVerifier.SignatureVerifier(),
+            //new DependencyAnalyzer.DependencyAnalyzer(),
```
Comment (cormacpayne) [inline tools/StaticAnalysis/Program.cs]: Will revert this change

File: tools/VersionController/Models/VersionMetadataHelper.cs [hunk 3]
```diff
@@ -302,7 +302,7 @@ public Version GetVersionBumpUsingSerialized(bool serialize = true)
                     CheckBreakingChangesInModules(oldModuleMetadata, newModuleMetadata, issueLogger);
                     if (issueLogger.Records.Any())
                     {
-                        tempVersionBump = Version.MAJOR;
+                        tempVersionBump = Version.PATCH;
```
Comment (cormacpayne) [inline tools/VersionController/Models/VersionMetadataHelper.cs]: Will revert this change

File: src/ResourceManager/Aks/Commands.Aks/AzureRM.Aks.psd1 [hunk 4]
```diff
@@ -51,12 +51,12 @@ CLRVersion = '4.0'
...
-RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.0'; })
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.1'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\YamlDotNet.dll',
-	'.\Microsoft.Azure.Commands.Common.Graph.RBAC.dll',
-	'.\AutoMapper.dll', '.\System.Management.Automation.dll'
+RequiredAssemblies = '.\YamlDotNet.dll', 
+               '.\Microsoft.Azure.Commands.Common.Graph.RBAC.dll', 
+               '.\AutoMapper.dll', 'System.Management.Automation.dll'
```
Comment (cormacpayne) [inline src/ResourceManager/Aks/Commands.Aks/AzureRM.Aks.psd1]: Will revert this change to `System.Management.Automation.dll`

**Comment by maddieclayton** [comment]
Only comment is regarding the version of sql discussed offline

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/811

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/813/
